extensions/extensions-docker/aissemble-quarkus/src/main/resources/scripts/run-env.sh

```diff
@@ -18,21 +18,21 @@ then
     then
       # Default to value that was originally in Dockerfile to maintain backwards compatability.
       echo "Defaulting KRAUSENING_BASE to /deployments/krausening/"
-      export JAVA_OPTIONS="$JAVA_OPTIONS -DKRAUSENING_BASE=/deployments/krausening/"
+      export JAVA_OPTS_APPEND="$JAVA_OPTS_APPEND -DKRAUSENING_BASE=/deployments/krausening/"
```

Comment:
I: The `run-java.sh` script in the base image uses this variable to set the java options instead of `JAVA_OPTIONS`. This allows it to have some more advanced features like a debug flag that automatically includes the right options at startup.